Office hours subpage (#147)
PrestaShop/open-source: content/office-hours/_index.md

---
title: Office hours

menu:
  main:
    parent: 'Community'
    name: 'Office hours'
---

# PrestaShop Project's Office Hours

> The Office Hours initiative is about project members making themselves available to the community (almost) every Tuesday at [11 am (Paris time)](https://time.is/1100_in_Paris), to discuss ongoing projects, pull requests, architectural decisions, etc. This is also a chance to answer some community questions and help them finish their contributions.

Reviewer: ```suggestion
The Office Hours initiative is about project members making themselves available to the community (almost) every Tuesday at [11 am (Paris time)](https://time.is/1100_in_Paris), to discuss ongoing projects, pull requests, architectural decisions, etc. This is also a chance to answer questions from the community and help contributors finish their contributions.
```

---
title: Office hours

menu:
  main:
    parent: 'Community'
    name: 'Office hours'
---

# PrestaShop Project's Office Hours

> The Office Hours initiative is about project members making themselves available to you, to the community, (almost) every Tuesday at 11 AM CET, to discuss everything related to the project.

Reviewer: ```suggestion
The Office Hours initiative is about project members making themselves available to the community (almost) every Tuesday at [11 am (Paris time)](https://time.is/1100_in_Paris), to discuss everything related to the project.
```

> These meetings have already proven they were a great idea. There have been constructive discussions about many ongoing projects, pull requests, architectural decisions, etc. This is also a chance to answer some community questions and help them finish their contributions.

Reviewer: I think it'd be best to keep the tone to a more "matter-of-fact", less opinionated style: what kinds of topics are discussed, why would it be interesting for me to participate?

Author: @eternoendless let me know if the new version is better

## Join the next meeting!

Would you like to discuss a pull request you are working on? Or the future of the project? Do you just want to listen to the discussion? If so, joining the meeting will be a great idea.

> Feel free to join on [this Google Meet link](https://meet.google.com/fer-hcgk-wti). Remember that Office Hours are held every Tuesday (except bank holiday days in most European countries) from 11:00 AM to 12 PM CET.

Reviewer: ```suggestion
Office Hours are held on [this Google Meet room](https://meet.google.com/fer-hcgk-wti) every Tuesday at [11 am (Paris time)](https://time.is/1100_in_Paris), except for bank holiday days in most European countries.
```

> If you want to suggest topics for discussion, please [fill out this form](https://forms.gle/2r7jPp5WxhFSUQme8) (it is not required, though!).

Reviewer: ```suggestion
If you want to suggest topics for discussion, please consider [filling out this form](https://forms.gle/2r7jPp5WxhFSUQme8).
```

---
title: Office hours

menu:
  main:
    parent: 'Community'
    name: 'Office hours'
    weight: 100
---

> # PrestaShop Project's Office hours

Reviewer 2: ```suggestion
# PrestaShop Project's Office Hours
```

The Office Hours initiative is about project members making themselves available to you, to the community, (almost) every Tuesday at 11 AM CET, to discuss everything related to the project.

> These meetings have already proven it was a great idea. There have been constructive discussions about many ongoing projects, Pull Requests, architectural decisions, etc. This is also a chance to answer some community questions and help them finish their contributions.

Reviewer 2: ```suggestion
These meetings have already proven they were a great idea. There have been constructive discussions about many ongoing projects, pull requests, architectural decisions, etc. This is also a chance to answer some community questions and help them finish their contributions.
```

## Join the next meeting!

> Would you like to discuss a Pull Request you are working on? Or the future of the project? Do you just want to listen to the discussion? If so, joining the meeting will be a great idea.

Reviewer 2: ```suggestion
Would you like to discuss a pull request you are working on? Or the future of the project? Do you just want to listen to the discussion? If so, joining the meeting will be a great idea.
```

> Feel free to join on [this Google Meet link](https://meet.google.com/fer-hcgk-wti). Remember that Office Hours are held every Tuesday (except days that are bank holidays in most European countries) from 11:00 AM to 12 PM CET.

Reviewer 2: ```suggestion
Feel free to join on [this Google Meet link](https://meet.google.com/fer-hcgk-wti). Remember that Office Hours are held every Tuesday (except bank holiday days in most European countries) from 11:00 AM to 12 PM CET.
```
